napari/layers/_base_layer/view/properties.py

```diff
@@ -117,17 +129,20 @@ def setExpanded(self, bool):
         if bool:
             self.expanded = True
             rows = self.grid_layout.rowCount()
-            self.setFixedHeight(55*(rows-1))
+            self.setFixedHeight(60*(rows-1) - 50)
         else:
             self.expanded = False
-            self.setFixedHeight(55)
+            self.setFixedHeight(60)
         rows = self.grid_layout.rowCount()
+        columns = self.grid_layout.columnCount()
         for i in range(1, rows):
-            for j in range(2):
-                if self.expanded:
-                    self.grid_layout.itemAtPosition(i, j).widget().show()
-                else:
-                    self.grid_layout.itemAtPosition(i, j).widget().hide()
+            for j in range(columns):
+                item = self.grid_layout.itemAtPosition(i, j)
+                if item is not None:
+                    if self.expanded:
```

Comment:
looking at this code now, and taking into consideration @jni's comments about left-alignment and layer name space, I would structure the layout like so:
```
VBoxLayout
    layer visibility | thumbnail | name
    FormLayout
        property name | property widget
```
then you can hide the entire form layout at once instead of iterating over and hiding each individual property like this

Reply:
fixed